_includes/platformcards.html

```diff
@@ -1,60 +1,22 @@
-        <div class="col">
-            <div class="card">
-                <div class="card-header">
-                    <h4 class="pt-2 text-center">Windows</h4>
-                </div>
-                <div class="d-flex justify-content-center">
-                    <img class="pt-4 pb-2" height="{{ imageSize }}" src="{% link assets/images/platform/Windows_logo_-_2012_derivative.svg %}"/>
-                </div>
-                <div class="card-body d-flex flex-column mx-1">
-                    <a class="btn btn-primary my-1" href="{% link _wiki/Install/Windows.md %}">
-                        Installation Guide
-                    </a>
-                    <a class="btn btn-secondary my-1" href="{% link _wiki/FAQ/Windows.md %}">
-                        FAQ
-                    </a>
-                </div>
-            </div>
-        </div>
-        <div class="col">
-            <div class="card">
-                <div class="card-header">
-                    <h4 class="pt-2 text-center">Linux</h4>
-                </div>
-                <div class="d-flex justify-content-center">
-                    <img class="pt-4 pb-2" height="{{ imageSize }}" src="{% link assets/images/platform/TuxFlat.svg %}"/>
-                </div>
-                <div class="card-body d-flex flex-column mx-1">
-                    <a class="btn btn-primary my-1" href="{% link _wiki/Install/Linux.md %}">
-                        Installation Guide
-                    </a>
-                    <a class="btn btn-secondary my-1" href="{% link _wiki/FAQ/Linux.md %}">
-                        FAQ
-                    </a>
-                </div>
-            </div>
-        </div>
-        <div class="col">
-            <div class="card">
-                <div class="card-header">
-                    <h4 class="pt-2 text-center">MacOS</h4>
-                </div>
-                <div class="d-flex justify-content-center">
-                    <img class="pt-4 pb-2" height="{{ imageSize }}" src="{% link assets/images/platform/Apple_Computer_Logo_rainbow.svg %}"/>
-                </div>
-                <div class="card-body d-flex flex-column mx-1">
-                    <a class="btn btn-primary my-1" href="{% link _wiki/Install/MacOS.md %}">
-                        Installation Guide
-                    </a>
-                    <a class="btn btn-secondary my-1" href="{% link _wiki/FAQ/MacOS.md %}">
-                        FAQ
-                    </a>
-                </div>
-            </div>
-        </div>
-    </div>
+  {% capture platformImage %}
+  {% link assets/images/platform/TuxFlat.svg %}
+  {% endcapture %}
+  {% include platformcard.html
+    platform="Linux"
+    platformImage=platformImage %}
+
+  {% capture platformImage %}
+  {% link assets/images/platform/Apple_Computer_Logo_rainbow.svg %}
+  {% endcapture %}
+  {% include platformcard.html
+    platform="MacOS"
+    platformImage=platformImage %}
```

Comment:
If we don't care about keeping `{% link %}` usage intact (as solving #67 will make sure we can't use bad paths) the 3 repetitions can be done over a data object (e.g. `/_data/os.json`) instead.